MIP41/mip41.md

```diff
@@ -149,11 +149,12 @@ If a Core Unit has no active Facilitators as a result of involuntary offboarding
 
 #### The Fallback Decision Process
 
-For the first fallback vote, a ranked choice vote from a list of volunteer Interim Facilitators and current active Facilitators is put up for one week. A simple majority vote on one candidate designates the Interim Facilitator. Should the first fallback vote fail, then the same list is put to all currently active Facilitators who choose to vote on a shorter time frame. 48 hours is recommended.
+For the first fallback vote, a ranked choice vote from a list of volunteer Interim Facilitators and current active Facilitators is put up for one week. The winner is the Interim Facilitator. In the case of a tie, the winner is selected by coin toss of an American 1964 silver quarter (or an otherwise fair coin). If the vote fails because nobody voted then all Facilitators shall endeavour to meet in person on Awashima Island, sit in a circle in a comfortable grassy field, and together commit ritual suicide (切腹).
```

Comment:
Note that having all Facilitators commit 切腹 would trigger the fallback decision process for all Core Units, which ---with all Facilitators gone--- would fail. This would spell disaster for the DAO. Is this meant to be a strong incentive for Facilitators to vote? Bug or feature? Discuss.

Reply:
[Well](https://media0.giphy.com/media/j9djzcMmzg8ow/giphy.gif?cid=ecf05e47wpq2mtwjqzoe3311ribeahdfkc9y770lpxzgpdzr&rid=giphy.gif&ct=g), it's certainly one way to test if we've automated ourselves out of a job.